elasticsearch/Dockerfile.in

```
@@ -59,7 +59,7 @@ ENV ES_PATH_CONF=/etc/elasticsearch/ \
 # WORKDIR $REMOTE_SOURCE_DIR/app/elasticsearch
 
 RUN packages="java-${JAVA_VER}-openjdk-headless \
-              python36 \
+              python39 \
```

The `python-unversioned-command` package has replaced the `alternatives` call in RHEL9. I can only suggest half of the change in the Github UI: line 70 needs to be removed as well.

```suggestion
              python-unversioned-command \
              python3 \
```